webapp/app.py

```diff
@@ -63,6 +73,80 @@ def _get_examples():
     return examples
 
 
+def _make_github_request(endpoint):
+    # Passing a gh username as a header increases
+    # the API rate limit from 60 to 5000 per hour
+    headers = { "username": random.choice(team_members)["login"] }
```

Comment:
What's the practice in the team with such things? It feels bit weird to use team members quota for that.
Don't we have some API keys that we could use to call GH API?

Reply:
I agree it's weird - I was working from [the API docs](https://docs.github.com/en/free-pro-team@latest/rest/overview/resources-in-the-rest-api#authentication), which talks about basic auth and oauth tokens, both of which are tied to users, rather than organisations. I couldn't see an organisation-based solution. 

The reason I went with basic auth is we'd have to store the OAuth token as a secret somewhere, which felt like it would make this nice-to-have feature more complex - but maybe it's easier than I thought?

Reply:
You should be [generating an access-token](https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/creating-a-personal-access-token) within our service account and authenticate using that(read it from an env var). You can test by creating one under your account first(read-only permissions should be enough for this), and then we can create one for staging and production. To authenticate you pass the token in the headers of the request `Authorization: token <generate-token>`